### PR TITLE
Update master branch packaging to harvest latest release of 1.1.0

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
* This updates the harvested S.P.SM assembly for netcore50 and netstandard1.3 to the most recently released version that was released in package 4.3.1
* The harvested S.P.SM assembly version should now be 4.1.1.1

### CODE REVIEW

**Comment by StephenBonikowsky** [comment]
I verified locally that version 4.1.1.1 of S.P.SM is now packaged under \runtimes\win7\lib for both netstandard1.3 and netcore50.

**Comment by StephenBonikowsky** [comment]
If this is good, I'll cherry-pick it to the release/2.1.0 branch

File: pkg/baseline/packageIndex.json [hunk 1]
```diff
@@ -16,12 +16,14 @@
       "StableVersions": [
         "4.0.0",
         "4.1.0",
-        "4.3.0"
+        "4.3.0",
```
Comment (StephenBonikowsky) [inline pkg/baseline/packageIndex.json]: @ericstj Not sure if I needed to keep 4.3.0 or simply override it with 4.3.1.

Comment (ericstj) [reply pkg/baseline/packageIndex.json]: Never delete versions from the index.  It's meant to be a private copy of package history and mirror nuget.org releases.

Comment (zhenlan) [reply pkg/baseline/packageIndex.json]: +1

Comment (StephenBonikowsky) [reply pkg/baseline/packageIndex.json]: Ok, thanks. I'll amend this PR.

**Pull request merged**